libs/eo/emissions/shell/src/lib/eo-emissions-page-co2-reduction.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-emissions-page-co2-reduction',
+  styles: [
+    `
+      :host {
+        height: 128px; // Magic number by designer
+        outline: 1px solid #f9d557; // Does not exist as a variable in our design system
+        display: grid;
+        grid-template-columns: 128px 1fr; // Magic numbers by designer
+
+        img {
+          width: 128px; // Magic number by designer
+          height: 128px; // Magic number by designer
+        }
+      }
+    `,
+  ],
+  template: `
+    <div>
```

Comment:
You could consider using the eo-media element here instead of using div and img :) A suggestion is already in `eo-origin-of-energy-global-goals-media.component.ts` :)

Reply:
This is going to sounds as Mr. Opposite.. But bear with me :-)

I will not use the eo media, in fact I think it is the biggest overkill/over engineering I have seen in years, in order to add an image placed next to some text. I would encourage you to remove this type of over engineering and instead move towards as easy maintainability code as possible.

But this is up to you.

Reply:
I am in agreement with you! :P